static/app/views/settings/organizationIntegrations/integrationDetailedView.tsx

```diff
@@ -242,54 +240,31 @@ class IntegrationDetailedView extends AbstractIntegrationDetailedView<
   renderTopButton(disabledFromFeatures: boolean, userHasAccess: boolean) {
     const {organization} = this.props;
     const provider = this.provider;
-    const {metadata} = provider;
-
-    const size = 'sm' as const;
-    const priority = 'primary' as const;
 
     const buttonProps = {
       style: {marginBottom: space(1)},
-      size,
-      priority,
+      size: 'sm' as const,
+      priority: 'primary' as const,
```

Comment:
TypeScript will infer the type, so you can just do:
```suggestion
      size: 'sm',
      priority: 'primary',
```